spec/models/form_profile_spec.rb

```diff
@@ -67,31 +67,6 @@
     }
   end
 
-  let(:v40_10007_expected) do
```

Comment:
Since there are still models for these two form profiles, these tests should probably be left in. If the forms are no longer used at all, it might be better to remove the models, tests and anything else relating to them in a separate PR.

Reply:
Hi @thilton-oddball, I pulled these tests out based on this [slack thread](https://dsva.slack.com/archives/CBU0KDSB1/p1642802354171800) to get around a `RSpec/MultipleMemoizedHelpers: Max: 37` error.  

What I would prefer to do is bump up the rubocop max and leave these tests alone.  Would you be alright with that?